ocaml-lsp-server/src/compl.ml

```diff
@@ -16,7 +16,7 @@ let make_string chars =
   let chars = Array.of_list chars in
   String.init (Array.length chars) ~f:(Array.get chars)
 
-let prefix_of_position source position =
+let prefix_of_position ?(short = false) source position =
```

Comment:
Let's make this argument mandatory and add a doc string to explain its purpose (or pick a better name perhaps?)

Reply:
If `short` is set, the prefix computation for `List.m<cursor>` returns just `m`; otherwise, it returns prefix `List.m`. I first called the param as `short_path` but opted for brevity. Both documenting or giving a more descriptive name (any ideas?) sound good to me.

Reply:
Both sound fine. `short_path` sounds clearer to me.